playground/imgui/imgui_raster.vert

```diff
@@ -9,13 +9,23 @@ uniforms;
 
 in vec2 vertex_position;
 in vec2 texture_coordinates;
-in uint vertex_color;
+in int vertex_color;
 
 out vec2 frag_texture_coordinates;
 out vec4 frag_vertex_color;
 
+vec4 ImVertexColorToVec4(int color) {
+  const float kScale = 1.0f / 255.0f;
+  return vec4(
+    ((color >> 0) & 0xFF)  * kScale,
+    ((color >> 8) & 0xFF)  * kScale,
+    ((color >> 16) & 0xFF) * kScale,
+    ((color >> 24) & 0xFF) * kScale
+  );
+}
+
 void main() {
   gl_Position = uniforms.mvp * vec4(vertex_position.xy, 0.0, 1.0);
   frag_texture_coordinates = texture_coordinates;
-  frag_vertex_color = unpackUnorm4x8(vertex_color);
```

Comment:
Ah yeah, thanks. I thought we might run into issues at some point with this weird builtin...